docker/README.md

```diff
@@ -35,4 +35,9 @@ See also [targets to run tests](../docs/dev/how-to-write-and-run-tests.md#runnin
 | `make clean`              | Clean up your dev environment: removes locally bound folders, run `hdown` and `prune`. | Run `make dev` to recreate a fresh dev env afterwards.        |
 | `make import_sample_data` | Load sample data (~100 products) into the MongoDB database.                            |                                                               |
 | `make import_prod_data`   | Load latest prod data (~2M products, 1.7GB) into the MongoDB database.                 | Takes up to 10m. Not recommended for dev setups !             |
+| `make lint_perltidy`      | Indent and reformat your code                         |
```

Comment:
maybe just "make lint" ? Easier to remember/type, and make lint is running make lint_perltidy

Reply:
Yeah, it is a good idea. I'll do this right now